Example/healthvault-ios-sdk/ViewControllers/MHVTypeViewController.m

```diff
@@ -349,7 +350,7 @@ - (void)removeCurrentThing
         
                            @synchronized (self.lockObject)
                            {
-                               [self.things removeObject:selectedThing];
+                               self.things = [self.things arrayByRemovingObject:selectedThing];
```

Comment:
We shouldn't be using the SDK extensions in the sample app. I will be making most of the extensions private - also, this method should probably go away.